ts/Series/PackedBubble/PackedBubbleSeries.ts

```diff
@@ -1151,6 +1167,26 @@ class PackedBubbleSeries extends BubbleSeries implements Highcharts.DragNodesSer
         return arr;
     }
 
+    public pointAttribs(
+        point?: PackedBubblePoint,
+        state?: StatesOptionsKey
+    ): SVGAttributes {
+        let options = this.options,
+            hasParentMarker = point && point.isParentNode,
+            markerOptions = hasParentMarker && options.layoutAlgorithm && options.layoutAlgorithm.parentNodeOptions ?
+                options.layoutAlgorithm.parentNodeOptions.marker :
+                options.marker,
+            fillOpacity = (markerOptions as any).fillOpacity,
```

Comment:
Do you know why we need the `any` casting in here? We should avoid it if possible. Maybe all we need to do is to simply change the TS type somewhere above?